recipes/libdivide/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools
+import os
+
+
+class LibdivideConan(ConanFile):
+    name = "libdivide"
+    description = "Header-only C/C++ library for optimizing integer division."
+    topics = ("conan", "libdivide", "division", "integer")
+    license = ["Zlib", "BSL-1.0"]
+    homepage = "http://libdivide.com/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "compiler"
```

Comment:
```suggestion
    settings = "compiler"
    options = {"libdivide_sse2": [True, False], "libdivide_avx2": [True, False], "libdivide_avx512": [True, False]}
    default_options = {"libdivide_sse2": False, "libdivide_avx2": False, "libdivide_avx512": False}
```

Reply:
I would name the options simple `sse2/avx2/avx512`. No need for the `libdivide_` prefix.

Removing these options when `self.settings.arch not in ("x86", "x86_64")` in `configure` is not necessary, I think, because this is a header-only package..

Reply:
seems like the upstream CMake handles the x86 https://github.com/ridiculousfish/libdivide/blob/0af3799ded543dabf9fca9b9214dfe297c2ed89e/CMakeLists.txt#L124

Seems like the options end up being mutually exclusive https://github.com/ridiculousfish/libdivide/blob/0af3799ded543dabf9fca9b9214dfe297c2ed89e/CMakeLists.txt#L118-L123

So maybe this?

```suggestion
    settings = "compiler"
    options = {"vector_instruction": [None, "sse2", "avx2", "avx512"]}
    default_options = {"vector_instruction": None }
```

Reply:
Other packages call this option `simd_intrinsics`, so maybe stick to that?

Reply:
I think it's easier for consumers too see the settings from the project 🤔 but that's a good suggestion

Reply:
I agree that it should be only one option. When avx512 is enabled, it doesn't change anything to enable or not sse2 or avx2. Same for avx2, doesn't matter to enable sse2 or not.

Reply:
addressed in https://github.com/conan-io/conan-center-index/pull/3315/commits/b42f1247d8264b444094cdd93f0ac1fac6df3a3f